Update README.md (#16)
DevExpress/devextreme-react-template: README.md

# DevExtreme React Template

> The DevExtreme React Template is a React application with a navigation menu and several sample views in a responsive layout (see a [live preview](https://devexpress.github.io/devextreme-react-template)). This application is created with [Create React App](https://create-react-app.dev/) and uses [DevExtreme React components](https://js.devexpress.com/Documentation/Guide/React_Components/DevExtreme_React_Components/).

Reviewer: ```suggestion
The DevExtreme React Template is a React application with a navigation menu and sample views in a responsive layout (see a [live preview](https://devexpress.github.io/devextreme-react-template)). This application is created with [Create React App](https://create-react-app.dev/) and uses [DevExtreme React components](https://js.devexpress.com/Documentation/Guide/React_Components/DevExtreme_React_Components/).
```

![DevExtreme-React-Template](https://user-images.githubusercontent.com/2280467/74529812-a8af1c80-4f39-11ea-96fb-efd94d19dbab.png)

## Getting Started

> For full information about the DevExtreme React Template and how to customize it, refer to the following help topic: [Create a DevExtreme Application](https://js.devexpress.com/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/).

Reviewer: ```suggestion
For more information about the DevExtreme React Template and how to customize it, refer to the following help topic: [Create a DevExtreme Application](https://js.devexpress.com/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/).
```

## License

**DevExtreme React Template is released as a MIT-licensed (free and open-source) add-on to DevExtreme.**

> Familiarize yourself with the [DevExtreme License](https://js.devexpress.com/Licensing/). [Free trial is available](http://js.devexpress.com/Buy/).

Reviewer: ```suggestion
More information about the [DevExtreme License](https://js.devexpress.com/Licensing/). [Free trial is available](http://js.devexpress.com/Buy/).
```

# DevExtreme React Template

The DevExtreme React Template allows you to generate an application with a navigation menu and several sample views in a responsive layout (see a [live preview](https://devexpress.github.io/devextreme-react-template)). You can customize the generated application as you wish.

![DevExtreme-React-Template](https://user-images.githubusercontent.com/2280467/74529812-a8af1c80-4f39-11ea-96fb-efd94d19dbab.png)

> The DevExtreme React Template is created with [Create React App](https://github.com/facebook/create-react-app) and uses [DevExtreme React components](https://js.devexpress.com/Documentation/Guide/React_Components/DevExtreme_React_Components/).

Reviewer 2: ```suggestion
The DevExtreme React Template is created with [Create React App](https://create-react-app.dev/) and uses [DevExtreme React components](https://js.devexpress.com/Documentation/Guide/React_Components/DevExtreme_React_Components/).
```

How about using another link for Create React App ?
